### ISSUE TITLE
Error with Mustache directive on UTF-8 Json file

### ISSUE THREAD

**Issue by ngirard** [CDSoft/pp#74]
Using latest `pp`.

Create `test.json` containing
```json
{ "attr": "é" }
```

and `test.md` containing
```
!mustache(test.json)
\```````````````````````````````````````
{{attr}}
\```````````````````````````````````````
```

`pp -html < test.md` fails with the following error message:
```
pp: Mustache (test.json): Error in $: Failed reading: Cannot decode input: Data.Text.Internal.Encoding.decodeUtf8: Invalid UTF-8 stream
```

**Comment by ngirard** [comment]
On-topic: https://stackoverflow.com/questions/27669418

Two people advised using `encodeUtf8 :: Text -> ByteString` from `Data.Text.Encoding` instead of `pack` from `Data.ByteString.Lazy.Char8`

**Comment by CDSoft** [comment]
Thanks. This fixes the bug. I'll release a new version soon.

**Comment by ngirard** [comment]
Great, thanks !

**Comment by CDSoft** [comment]
Fixed in 2.7.3.
